src/averageAges.js

```diff
@@ -36,8 +45,18 @@ function calculateMenAverageAge(people, century) {
  *
  * @return {number}
  */
-function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+function calculateWomenAverageAge(people, withChildren = false) {
+  const woomen = people.filter(person => person.sex === femaleSex);
+  const filteredWoomen = withChildren !== false ? woomen.filter(woman =>
```

Comment:
Why is `withChildren` checked twice here? One check in the middle of the `filter` is enough.